openeo/capabilities.py

```diff
@@ -78,11 +103,21 @@ class ComparableVersion:
         `a.accept_lower(b)`: b is lower than a
     """
 
+    component_re = re.compile(r'(\d+ | [a-z]+ | \.)', re.VERBOSE)
+
     def __init__(self, version: Union[str, 'ComparableVersion']):
         if isinstance(version, ComparableVersion):
             self._version = version._version
         else:
-            self._version = LooseVersion(version)
+            self.parse(version)
+
+    def parse(self, version_string):
```

Comment:
I think this should be a (private) class method, that doesn't modify `self`, but just returns a tuple